examples/src/index.tsx

```diff
@@ -2,8 +2,6 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
 
-import './index.css';
-
```

Comment:
I decided to move this inline in the `index.html` page to remove some complexity

Reply:
Sounds good :+1: